mlrun/feature_store/steps.py

```diff
@@ -166,9 +167,15 @@ def _encode(self, feature: str, value):
         encoding = self.mapping.get(feature, [])
 
         if encoding:
-            one_hot_encoding = {f"{feature}_{category}": 0 for category in encoding}
+
+            one_hot_encoding = {
+                f"{feature}_{OneHotEncoder._sanitized_category(category)}": 0
+                for category in encoding
+            }
             if value in encoding:
-                one_hot_encoding[f"{feature}_{value}"] = 1
+                one_hot_encoding[
+                    f"{feature}_{OneHotEncoder._sanitized_category(value)}"
```

Comment:
Does it actually make sense to pass `value` to `_sanitized_category`? (since afaict it's not a category...)

Reply:
f"{feature}_{value}" is a category of the new dataset after the ingestion, so I can pass all of it if it make more sense for you? @gtopper